blankly/strategy/strategy_base.py

```diff
@@ -115,35 +120,41 @@ def __idle_event(self):
         
     def __price_event_rest(self, **kwargs):
         callback = kwargs['callback']
-        currency_pair = kwargs['currency_pair']
+        symbol = kwargs['symbol']
         resolution = kwargs['resolution']
         variables = kwargs['variables']
+        ohlc = kwargs['ohlc']
         state = kwargs['state_object']  # type: StrategyState
-        price = self.Interface.get_price(currency_pair)
 
         state.variables = variables
         state.resolution = resolution
 
-        callback(price, currency_pair, state)
+        if ohlc:
+            data = self.Interface.history(symbol, 1, resolution).iloc[0].to_dict()
```

Comment:
This probably won't work correctly. You need to wait & check until the most recent download has the price that the round up function predicted.